draft-ietf-dnsop-svcb-https.md

```diff
@@ -1056,9 +1056,10 @@ differences in the intended client and server behavior.
 
 ### ALPN usage
 
-Unlike Alt-Svc Field Values, HTTPS RRs can contain multiple ALPN
-IDs, and clients are encouraged to offer additional ALPNs that they
-support.
+Unlike Alt-Svc Field Values, HTTPS RRs can contain multiple ALPN IDs.  Clients
+might choose to remove less-preferred ALPN IDs they offer when using HTTPS RRs,
+but they are not required to abort a connection attempt if a chosen ALPN ID is
+not negotiated by the server.
```

Comment:
```suggestion
Unlike Alt-Svc Field Values, HTTPS RRs can contain multiple ALPN IDs.
```

In my view, the text on ALPNs in RFC 7838 is ambiguous, and I would like to avoid attempting to clarify it in this draft.  (We can leave that for alt-svc-bis.)  I don't think "the client is required to abort if the indicated ALPN is not selected" is the only plausible reading of RFC 7838, and it's not the interpretation I prefer.

Since we don't seem to have agreement on the actual distinction between these drafts, I think we're best off simply removing this explanatory text.

Reply:
I'm fine with Ben's latest proposal.  Martin?

Reply:
Yes, I think that Ben nailed it.